### PR TITLE
Add Pila87/XM87 Pro

### PR DESCRIPTION
**Pull request by Pangorin**
## Description

Add support for the Pila87/XM87 Pro

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

Add support for the Pila87/XM87 Pro

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by Pangorin — COMMENTED** [review]
Delete some old files

File: keyboards/pangorin/pila87_mod/config.h [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x5048 //Phage Studio
+#define PRODUCT_ID      0x5887
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Phage Studio
+#define PRODUCT         Pila87
```
Comment (drashna) [inline keyboards/pangorin/pila87_mod/config.h:27]: Please move these values to the info.json file.

File: keyboards/pangorin/pila87_mod/config.h [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+/* Key matrix pin          0   1,   2,  3,  4,  5,  6,  7,  8,  9,  10,  11,  12,  13,  14,  15, 16 */
+#define MATRIX_ROW_PINS { A1, A2, A10, A9, A8, C9 }
+#define MATRIX_COL_PINS { B5, B4,  B6, C0, C1, C2, C3, B7, B3, D2, C12, C11, C10, A15, C15, C13, A3 }
+
+/* Direction of diode (COL2ROW or ROW2COL) */
+#define DIODE_DIRECTION COL2ROW
+
+/* Forcing to use NKRO instead 6KRO */
+#define FORCE_NKRO
```
Comment (drashna) [inline keyboards/pangorin/pila87_mod/config.h:41]: This can cause issues on some systems.

File: keyboards/pangorin/pila87_mod/config.h [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+/* Change USB Polling Rate to 1000hz and a larger keys per scan for elite gaming */
+#define USB_POLLING_INTERVAL_MS 1
```
Comment (drashna) [inline keyboards/pangorin/pila87_mod/config.h:44]: this is the default behavior already. 
```suggestion
```

File: keyboards/pangorin/pila87_mod/config.h [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+#define QMK_KEYS_PER_SCAN 12
```
Comment (drashna) [inline keyboards/pangorin/pila87_mod/config.h:45]: deprecated. 
```suggestion
```

File: keyboards/pangorin/pila87_mod/rules.mk [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/pangorin/pila87_mod/rules.mk:18]: Please remove the `SLEEP_LED_ENABLE` setting, as it's not really used, and remove the comments for both it and NKRO.  
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

**Review by drashna — CHANGES_REQUESTED** [review]
the animation c files need licenses headers, too

**Review by Pangorin — COMMENTED** [review]
Delete some unnecessary things.

**Review by Pangorin — COMMENTED** [review]
Edit some unnecessary things.

**Review by Pangorin — COMMENTED** [review]
Delete things that will cause issues on some systems.

**Review by drashna — APPROVED** [review]

File: keyboards/pangorin/pila87_mod/readme.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+**Warning:**
+```
+Before using this QMK firmware, you need these thing below:
+- 1 Quartz Crystal (SMD 3225) 8 Mhz
+- 2 Capacitor 0603 18pF (or 22pF)
+- 1 ST-Link v2 to connect SWD and flash jumploader
+```
+Make example for this keyboard (after setting up your build environment):
+
+    make phage_studio/pila87_mod:default
```
Comment (zvecr) [inline keyboards/pangorin/pila87_mod/readme.md:27]:
```suggestion
    make pangorin/pila87_mod:default
```

File: keyboards/pangorin/pila87_mod/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+enum layer_names {
+    _BASE = 0,
+    _FN = 1
+};
+
+// enum layer_keycodes { };
```
Comment (zvecr) [inline keyboards/pangorin/pila87_mod/keymaps/default/keymap.c:29]:
```suggestion
enum layer_names {
    _BASE,
    _FN
};
```

File: keyboards/pangorin/pila87_mod/config.h [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+#define RGB_MATRIX_KEYPRESSES
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#define DISABLE_RGB_MATRIX_BAND_SAT
+#define DISABLE_RGB_MATRIX_BAND_PINWHEEL_SAT
+#define DISABLE_RGB_MATRIX_BAND_SPIRAL_SAT
+#define DISABLE_RGB_MATRIX_SOLID_REACTIVE_WIDE
+#define DISABLE_RGB_MATRIX_SOLID_REACTIVE_CROSS
+#define DISABLE_RGB_MATRIX_SOLID_REACTIVE_NEXUS
+#define DISABLE_RGB_MATRIX_SPLASH
+#define DISABLE_RGB_MATRIX_SOLID_SPLASH
```
Comment (zvecr) [inline keyboards/pangorin/pila87_mod/config.h:62]: These are not valid config options. Effects are now opt-in, <https://docs.qmk.fm/#/feature_rgb_matrix?id=rgb-matrix-effects>.

File: keyboards/pangorin/pila87_mod/keymaps/via/keymap.c [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+enum layer_names {
+    _BASE = 0,
+    _FN = 1
+};
+
+// enum layer_keycodes { };
```
Comment (zvecr) [inline keyboards/pangorin/pila87_mod/keymaps/via/keymap.c:29]:
```suggestion
enum layer_names {
    _BASE,
    _FN
};
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by Pangorin — COMMENTED** [review]
Commit changes and update

**Review by Pangorin — COMMENTED** [review]
Change RGB_MATRIX codes

File: keyboards/pangorin/pila87_mod/pila87_mod.c [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "pila87_mod.h"
+
+led_config_t g_led_config = { {
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/pila87_mod.c:19]:
```suggestion
#ifdef RGB_MATRIX_ENABLE
led_config_t g_led_config = { {
```

File: keyboards/pangorin/pila87_mod/pila87_mod.c [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+    {224, 17}, {211, 17}, {198, 17}, {175, 17}, {156, 17}, {143, 17}, {130, 17}, {117, 17}, {104, 17}, {91 , 17}, {78 , 17}, {65 , 17}, {52 , 17}, {39 , 17}, {26 , 17}, {13 , 17}, {0  , 17},
+    {0  ,  0}, {26 ,  0}, {39 ,  0}, {52 ,  0}, {65 ,  0}, {84 ,  0}, {97 ,  0}, {110,  0}, {123,  0}, {143,  0}, {156,  0}, {169,  0}, {182,  0}, {198,  0}, {211,  0},            {224,  0},
+}, {
+   4, 4, 4, 1, 1, 1, 1,       4, 4, 4, 4, 4, 1, 1, 1,
+      1,    4,       4, 4, 4, 4, 4, 4, 4, 4, 4, 1, 4,
+            1,    4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 8,
+   1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+   4, 4, 4, 1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+   4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 8,    4,
+} };
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/pila87_mod.c:40]:
```suggestion
} };
#endif
```

File: keyboards/pangorin/pila87_mod/keymaps/via/keymap.c [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+       │      │Ins│Del│Hom│End│PgU│PgD│ ← │ ↓ │ → │   │   │        │
+       ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤     ┌───┐
+       │        │   │   │   │   │   │   │   │   │   │   │          │     │   │
+       ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤ ┌───┼───┼───┐
+       │    │    │    │                        │    │ Fn │    │    │ │   │   │   │
+       └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘ └───┴───┴───┘
+*/
+    /*  Row:    0        1        2        3        4        5        6        7        8        9         10       11       12       13       14       15       16     */
+    [_FN]   = LAYOUT_tkl_ansi(
+                RESET,            RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD,  RGB_SPI, RGB_SPD, _______, NK_TOGG, _______, _______, _______,
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/keymaps/via/keymap.c:89]:
```suggestion
                QK_BOOT,          RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD,  RGB_SPI, RGB_SPD, _______, NK_TOGG, _______, _______, _______,
```

File: keyboards/pangorin/pila87_mod/keymaps/via/keymap.c [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+                _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,    KC_F10,  KC_F11,  KC_F12,  _______, _______, _______, _______,
+                _______, _______, KC_BSPC, _______, _______, _______, _______, _______, KC_UP,   _______,  _______, _______, _______, _______, _______, _______, _______,
+                _______, KC_INS,  KC_DEL,  KC_HOME, KC_END,  KC_PGUP, KC_PGDN, KC_LEFT, KC_DOWN, KC_RIGHT, _______, _______,          _______,
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/keymaps/via/keymap.c:92]:
```suggestion
                _______, KC_INS,  KC_DEL,  KC_HOME, KC_END,  KC_PGUP, KC_PGDN, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______,          _______,
```

File: keyboards/pangorin/pila87_mod/keymaps/via/keymap.c [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+       │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ : │ " │        │
+       ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤     ┌───┐
+       │ LShift │ Z │ X │ C │ V │ B │ N │ M │ < │ > │ ? │   RShift │     │   │
+       ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤ ┌───┼───┼───┐
+       │    │    │    │                        │    │    │    │    │ │   │   │   │
+       └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘ └───┴───┴───┘
+*/
+    /*  Row:    0        1        2        3      4      5       6      7      8        9       10       11       12       13       14       15       16      */
+    [_BASE] = LAYOUT_tkl_ansi(
+                KC_ESC,           KC_F1,   KC_F2, KC_F3, KC_F4,  KC_F5, KC_F6, KC_F7,   KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PSCR, KC_SLCK, KC_PAUSE,
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/keymaps/via/keymap.c:63]:
```suggestion
                KC_ESC,           KC_F1,   KC_F2, KC_F3, KC_F4,  KC_F5, KC_F6, KC_F7,   KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PSCR, KC_SCRL, KC_PAUS,
```

File: keyboards/pangorin/pila87_mod/keymaps/via/keymap.c [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+                KC_GRV,  KC_1,    KC_2,    KC_3,  KC_4,  KC_5,   KC_6,  KC_7,  KC_8,    KC_9,   KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_INS,  KC_HOME, KC_PGUP,
+                KC_TAB,  KC_Q,    KC_W,    KC_E,  KC_R,  KC_T,   KC_Y,  KC_U,  KC_I,    KC_O,   KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL,  KC_END,  KC_PGDN,
+                KC_CAPS, KC_A,    KC_S,    KC_D,  KC_F,  KC_G,   KC_H,  KC_J,  KC_K,    KC_L,   KC_SCLN, KC_QUOT,          KC_ENT,
+                KC_LSFT, KC_Z,    KC_X,    KC_C,  KC_V,  KC_B,   KC_N,  KC_M,  KC_COMM, KC_DOT, KC_SLSH,                   KC_RSFT,          KC_UP,
+                KC_LCTL, KC_LGUI, KC_LALT,               KC_SPC,                                KC_RALT, MO(_FN), KC_APP,  KC_RCTL, KC_LEFT, KC_DOWN, KC_RIGHT
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/keymaps/via/keymap.c:68]:
```suggestion
                KC_LCTL, KC_LGUI, KC_LALT,               KC_SPC,                                KC_RALT, MO(_FN), KC_APP,  KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
```

File: keyboards/pangorin/pila87_mod/keymaps/horrortroll/keymap.c [hunk 16]
```diff
@@ -0,0 +1,88 @@
...
+       │      │2Hd│2Hi│2Sd│2Si│2Vd│2Vi│   │   │   │   │   │        │
+       ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤     ┌───┐
+       │        │   │Pre│Ref│Flp│   │   │   │   │   │   │          │     │Vai│
+       ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤ ┌───┼───┼───┐
+       │    │    │    │                        │    │ Fn │    │    │ │Hud│Vad│Hui│
+       └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘ └───┴───┴───┘
+*/
+    /*  Row:    0        1        2        3        4        5        6        7        8        9        10       11       12       13       14        15       16     */
+    [_FN]   = LAYOUT_tkl_ansi(
+                RESET,            KC_MSEL, KC_VOLD, KC_VOLU, KC_MUTE, KC_MSTP, KC_MPRV, KC_MPLY, KC_MNXT, KC_MAIL, KC_WHOM, KC_CALC, KC_WSCH, RGB_RMOD, RGB_MOD, RGB_TOG,
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/keymaps/horrortroll/keymap.c:81]:
```suggestion
                QK_BOOT,          KC_MSEL, KC_VOLD, KC_VOLU, KC_MUTE, KC_MSTP, KC_MPRV, KC_MPLY, KC_MNXT, KC_MAIL, KC_WHOM, KC_CALC, KC_WSCH, RGB_RMOD, RGB_MOD, RGB_TOG,
```

File: keyboards/pangorin/pila87_mod/keymaps/horrortroll/keymap.c [hunk 17]
```diff
@@ -0,0 +1,88 @@
...
+       │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ : │ " │        │
+       ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤     ┌───┐
+       │ LShift │ Z │ X │ C │ V │ B │ N │ M │ < │ > │ ? │   RShift │     │   │
+       ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤ ┌───┼───┼───┐
+       │    │    │    │                        │    │    │    │    │ │   │   │   │
+       └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘ └───┴───┴───┘
+*/
+    /*  Row:    0        1        2        3      4      5       6      7      8        9       10       11       12       13       14       15       16      */
+    [_BASE] = LAYOUT_tkl_ansi(
+                KC_ESC,           KC_F1,   KC_F2, KC_F3, KC_F4,  KC_F5, KC_F6, KC_F7,   KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PSCR, KC_SLCK, KC_PAUSE,
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/keymaps/horrortroll/keymap.c:55]:
```suggestion
                KC_ESC,           KC_F1,   KC_F2, KC_F3, KC_F4,  KC_F5, KC_F6, KC_F7,   KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PSCR, KC_SCRL, KC_PAUS,
```

File: keyboards/pangorin/pila87_mod/keymaps/horrortroll/keymap.c [hunk 18]
```diff
@@ -0,0 +1,88 @@
...
+                KC_GRV,  KC_1,    KC_2,    KC_3,  KC_4,  KC_5,   KC_6,  KC_7,  KC_8,    KC_9,   KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_INS,  KC_HOME, KC_PGUP,
+                KC_TAB,  KC_Q,    KC_W,    KC_E,  KC_R,  KC_T,   KC_Y,  KC_U,  KC_I,    KC_O,   KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL,  KC_END,  KC_PGDN,
+                KC_CAPS, KC_A,    KC_S,    KC_D,  KC_F,  KC_G,   KC_H,  KC_J,  KC_K,    KC_L,   KC_SCLN, KC_QUOT,          KC_ENT,
+                KC_LSFT, KC_Z,    KC_X,    KC_C,  KC_V,  KC_B,   KC_N,  KC_M,  KC_COMM, KC_DOT, KC_SLSH,                   KC_RSFT,          KC_UP,
+                KC_LCTL, KC_LGUI, KC_LALT,               KC_SPC,                                KC_RALT, MO(_FN), KC_APP,  KC_RCTL, KC_LEFT, KC_DOWN, KC_RIGHT
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/keymaps/horrortroll/keymap.c:60]:
```suggestion
                KC_LCTL, KC_LGUI, KC_LALT,               KC_SPC,                                KC_RALT, MO(_FN), KC_APP,  KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
```

File: keyboards/pangorin/pila87_mod/keymaps/default/keymap.c [hunk 19]
```diff
@@ -0,0 +1,106 @@
...
+       │      │Ins│Del│Hom│End│PgU│PgD│ ← │ ↓ │ → │   │   │        │
+       ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤     ┌───┐
+       │        │   │   │   │   │   │   │   │   │   │   │          │     │   │
+       ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤ ┌───┼───┼───┐
+       │    │    │    │                        │    │ Fn │    │    │ │   │   │   │
+       └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘ └───┴───┴───┘
+*/
+    /*  Row:    0        1        2        3        4        5        6        7        8        9         10       11       12       13       14       15       16     */
+    [_FN]   = LAYOUT_tkl_ansi(
+                RESET,            RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD,  RGB_SPI, RGB_SPD, _______, NK_TOGG, _______, _______, _______,
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/keymaps/default/keymap.c:89]:
```suggestion
                QK_BOOT,          RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD,  RGB_SPI, RGB_SPD, _______, NK_TOGG, _______, _______, _______,
```

File: keyboards/pangorin/pila87_mod/keymaps/default/keymap.c [hunk 20]
```diff
@@ -0,0 +1,106 @@
...
+                _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,    KC_F10,  KC_F11,  KC_F12,  _______, _______, _______, _______,
+                _______, _______, KC_BSPC, _______, _______, _______, _______, _______, KC_UP,   _______,  _______, _______, _______, _______, _______, _______, _______,
+                _______, KC_INS,  KC_DEL,  KC_HOME, KC_END,  KC_PGUP, KC_PGDN, KC_LEFT, KC_DOWN, KC_RIGHT, _______, _______,          _______,
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/keymaps/default/keymap.c:92]:
```suggestion
                _______, KC_INS,  KC_DEL,  KC_HOME, KC_END,  KC_PGUP, KC_PGDN, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______,          _______,
```

File: keyboards/pangorin/pila87_mod/keymaps/default/keymap.c [hunk 21]
```diff
@@ -0,0 +1,106 @@
...
+       │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ : │ " │        │
+       ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤     ┌───┐
+       │ LShift │ Z │ X │ C │ V │ B │ N │ M │ < │ > │ ? │   RShift │     │   │
+       ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤ ┌───┼───┼───┐
+       │    │    │    │                        │    │    │    │    │ │   │   │   │
+       └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘ └───┴───┴───┘
+*/
+    /*  Row:    0        1        2        3      4      5       6      7      8        9       10       11       12       13       14       15       16      */
+    [_BASE] = LAYOUT_tkl_ansi(
+                KC_ESC,           KC_F1,   KC_F2, KC_F3, KC_F4,  KC_F5, KC_F6, KC_F7,   KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PSCR, KC_SLCK, KC_PAUSE,
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/keymaps/default/keymap.c:63]:
```suggestion
                KC_ESC,           KC_F1,   KC_F2, KC_F3, KC_F4,  KC_F5, KC_F6, KC_F7,   KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PSCR, KC_SCRL, KC_PAUS,
```

File: keyboards/pangorin/pila87_mod/keymaps/default/keymap.c [hunk 22]
```diff
@@ -0,0 +1,106 @@
...
+                KC_GRV,  KC_1,    KC_2,    KC_3,  KC_4,  KC_5,   KC_6,  KC_7,  KC_8,    KC_9,   KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_INS,  KC_HOME, KC_PGUP,
+                KC_TAB,  KC_Q,    KC_W,    KC_E,  KC_R,  KC_T,   KC_Y,  KC_U,  KC_I,    KC_O,   KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL,  KC_END,  KC_PGDN,
+                KC_CAPS, KC_A,    KC_S,    KC_D,  KC_F,  KC_G,   KC_H,  KC_J,  KC_K,    KC_L,   KC_SCLN, KC_QUOT,          KC_ENT,
+                KC_LSFT, KC_Z,    KC_X,    KC_C,  KC_V,  KC_B,   KC_N,  KC_M,  KC_COMM, KC_DOT, KC_SLSH,                   KC_RSFT,          KC_UP,
+                KC_LCTL, KC_LGUI, KC_LALT,               KC_SPC,                                KC_RALT, MO(_FN), KC_APP,  KC_RCTL, KC_LEFT, KC_DOWN, KC_RIGHT
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/keymaps/default/keymap.c:68]:
```suggestion
                KC_LCTL, KC_LGUI, KC_LALT,               KC_SPC,                                KC_RALT, MO(_FN), KC_APP,  KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
```

File: keyboards/pangorin/pila87_mod/info.json [hunk 23]
```diff
@@ -0,0 +1,103 @@
+{
+    "keyboard_name": "Pila87 / XM87 Pro",
```
Comment (fauxpark) [inline keyboards/pangorin/pila87_mod/info.json:2]: Need `manufacturer` entry here as well.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/pangorin/pila87_mod/info.json (hunk 23 above)
Comment (Pangorin) [reply keyboards/pangorin/pila87_mod/info.json:2]: I'll add it asap

**Review by Pangorin — COMMENTED** [review]
Fix literally everything by fauxpark :D

**Review by Pangorin — COMMENTED** [review]
Typo

**Review by Pangorin — COMMENTED** [review]
More typo :(

**Review by Pangorin — COMMENTED** [review]
Delete error codes

**Pull request closed without merging**